lang/markdown/markdown.talon

```diff
@@ -1,14 +1,49 @@
 tag: user.markdown
 -
-level one: "# "
-level two: "## "
-level three: "### "
-level four: "#### "
-level five: "##### "
-level six: "###### "
+(level | heading | header) one:
+    edit.line_start()
+    "# "
+(level | heading | header) two:
+    edit.line_start()
+    "## "
+(level | heading | header) three:
+    edit.line_start()
+    "### "
+(level | heading | header) four:
+    edit.line_start()
+    "#### "
+(level | heading | header) five:
+    edit.line_start()
+    "##### "
+(level | heading | header) six:
+    edit.line_start()
+    "###### "
+
+list [one]:
+    edit.line_start()
+    "- "
+list two:
+    edit.line_start()
+    "    - "
+list three:
+    edit.line_start()
+    "        - "
+list four:
+    edit.line_start()
+    "            - "
+list five:
+    edit.line_start()
+    "                - "
+list six:
+    edit.line_start()
+    "                    - "
 
 {user.markdown_code_block_language} block:
     "```{markdown_code_block_language}"
     key(enter:2)
     "```"
     key(up)
+
+link:
+    "[]()"
+    key(left:3)
```

Comment:
I think we prefer `user.insert_between` for this sort of thing, right?

Reply:
Yeah we probably do.